tests/Aspire.Components.Common.Tests/Aspire.Components.Common.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <TestingPlatformCommandLineArguments>$(TestingPlatformCommandLineArguments) --ignore-exit-code 8</TestingPlatformCommandLineArguments>
```

Comment:
```suggestion
    <TestingPlatformCommandLineArguments>$(TestingPlatformCommandLineArguments) --ignore-exit-code </TestingPlatformCommandLineArguments>
```
?

Reply:
The exit code 8 is "zero tests ran". Currently, all tests in this project seem to be ignored when running in CI. I'll add a link to https://learn.microsoft.com/en-us/dotnet/core/testing/microsoft-testing-platform-exit-codes above this line.